packages/bpk-component-phone-input/src/BpkPhoneInput.js

```diff
@@ -58,6 +58,20 @@ type CommonProps = {
   valid: ?boolean,
 };
 
+// This function get the size value of the dialingCode to resize the field correctly
+const widthForDialingCode = (large, dialingCodes, selectedDialingCode) => {
+  // This sizeConstant is an average character size for each size of field
+  const averageLetterSize = large ? 8 : 6;
+  const foundDialogCode = dialingCodes.find(
```

Comment:
Typo in the variable name 😇 

```suggestion
  const foundDialingCode = dialingCodes.find(
```